projects_test.go

```diff
@@ -70,7 +70,7 @@ func TestListUserProjects(t *testing.T) {
 	}
 }
 
-func ListProjectsUsers_byID(t *testing.T) {
+func ListProjectsUsersByID(t *testing.T) {
```

Comment:
If you want to fix them the right way, these functions should be called `TestList...`. And maybe (while your at it) also fix `TestGetProject_byID` and `TestGetProject_byName`?

Reply:
Well, I found `ListProjectsUsersByXx` not tested.

```
go test -v -run ListProject
=== RUN   TestListProjectPipelines
--- PASS: TestListProjectPipelines (0.00s)
=== RUN   TestListProjects
--- PASS: TestListProjects (0.00s)
=== RUN   TestListProjectForks
--- PASS: TestListProjectForks (0.00s)
PASS
ok      github.com/xanzy/go-gitlab      0.022s
```

After I And `Test` prefix to them, a **FAIL** occurred.

```
go test -v -run ListProject
=== RUN   TestListProjectPipelines
--- PASS: TestListProjectPipelines (0.00s)
=== RUN   TestListProjects
--- PASS: TestListProjects (0.00s)
=== RUN   TestListProjectsUsersByID
--- FAIL: TestListProjectsUsersByID (0.00s)
        projects_test.go:90: Projects.ListProjectsUsers returned error: GET http://127.0.0.1:64541/projects/1/users: 404 failed to parse unexpected error type: <nil>
        projects_test.go:95: Projects.ListProjectsUsers returned [], want [0xc4200703c0 0xc420070420]
=== RUN   TestListProjectsUsersByName
--- FAIL: TestListProjectsUsersByName (0.00s)
        gitlab_test.go:37: Request url: /projects/namespace%2Fname/users?page=2&per_page=3&search=query, want /projects/namespace%2Fname/users
=== RUN   TestListProjectForks
--- PASS: TestListProjectForks (0.00s)
FAIL
exit status 1
FAIL    github.com/xanzy/go-gitlab      0.027s
```

 I'll take a look later.